se_proj/edu_app/views.py

```diff
@@ -63,9 +63,30 @@ def login(request):
 
 
 def assignments(request):
-    return render(request, "assignments.html")
+    user_id = request.session.get('id')
+    
+    if not user_id:
```

Comment:
Couldn't say what the behavior of this condition is because I don't know python well enough. Is user_id a boolean?